src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -0,0 +1,112 @@
+
+   > Manually apply patches
+
+   ```bash
+   php ./vendor/bin/ece-patches apply
+   ```
+
+## Magento Cloud Docker changes
+
+-  **The minimum PHP version requirement is now PHP 7.1**–If your Magento Cloud Docker host is running an earlier version, upgrade to PHP v7.1 or later.
+
+-  **Magento Cloud Docker command changes**–
+
+   -  **Updating Magento Cloud Docker commands for Docker build operations**–We moved the Magento Cloud Docker commands from the `vendor/bin/ece-tools` directory to the `vendor/bin/ece-docker` directory. Update your scripts and commands to use the new path.
+
+      After upgrading to the {{site.data.var.ct}} 2002.1.0 or {{site.data.var.mcd}} 1.0.0, use the following command to view available `ece-docker` commands.
+
+      ```bash
+      php ./vendor/bin/ece-docker list
+      ```
+
+   -  **Updating the Magento Cloud docker-compose commands**–We renamed the path to the command file from `./bin/docker` to `./bin/magento-docker`.  Update your scripts and commands to use the new path.
+
+   -  **Using temporary containers**–In previous versions, the containers created by `bin/magento-docker` command operations were not removed, so you could use them for other operations.  Now, the `magento-docker` commands remove any containers they create after the command completes.
+
+      If you want to keep a container created by a docker-compose operation, use the `docker-compose run` command instead of the `bin/magento-docker` command.
+
+   -  **Running post-deploy hooks**–The `cloud-deploy` command no longer runs post deploy hooks. You must use the new `cloud-post-deploy` command to run post deploy hooks after you deploy.
+
+      ```bash
+      bin/magento-docker ece-deploy
+      ```
+
+      ```bash
+      bin/magento-docker ece-post-deploy
+      ```
+
+      Alternatively, if you use `docker-compose` commands directly, run the `docker-compose run deploy cloud-post-deploy` command after the deploy command.
+
+-  **Refreshing the database**–The Database container is now stored in a persistent Docker volume named `magento-db`. When you refresh the Docker environment, the database is no longer deleted automatically. If needed, use one of the following commands to manually remove it.
+
+   -  Remove the `magento-db` container:
+
+      ```bash
+      docker volume rm magento-db
+      ```
+
+   -  Remove all associated volumes when shutting down the Docker containers:
+
+      ```bash
+      docker-compose down -v
+      ```
+
+-  **Override file synchronization settings for archive and backup files**–Archive and backup files with the following extensions are no longer synchronized when using docker-sync or mutagen:  `*.sql`, `*.gz`, `*.zip`, and `*.bz2`.  You can override the default file synchronization for these file types by renaming the file to end with a different extension, for example `synchronize-me.zip-backup`.
```

Comment:
> for example `synchronize-me.zip-backup`.

This appears as if the period is part of the filename. It is tricky to end a sentence with something like this, and in this paragraph you have two. Consider:

```suggestion
-  **Override file synchronization settings for archive and backup files**–Archive and backup files with the following extensions are no longer synchronized when using docker-sync or mutagen:  `*.sql`, `*.gz`, `*.zip`, and `*.bz2`. You can override the default file synchronization for these file types by renaming the file to end with a different extension. For example: `synchronize-me.zip-backup`
```